Externals.cfg

```diff
@@ -1,9 +1,9 @@
 [regional_workflow]
 protocol = git
-repo_url = https://github.com/NOAA-EMC/regional_workflow
+repo_url = https://github.com/chan-hoo/regional_workflow
```

Comment:
Sorry for the late review here...please remove the changes to Externals.cfg, these should not be committed to master